mdpow/config.py

```diff
@@ -233,7 +240,7 @@ def get_configuration(filename=None):
 def modify_gromacs_environment(name, value):
     from gromacs.environment import flags
     if flags[name] != value:
-        logger.warn("Changing GromacsWrapper environment: flags[%(name)r] = %(value)r", vars())
+        logger.warning("Changing GromacsWrapper environment: flags[%(name)r] = %(value)r", vars())
```

Comment:
That won't work with 2.7. At the top of the file add after the logger definition
```python
logger = logging.getLogger(".....")
if not hasattr(logger, "warning"):
   logger.warning = logger.warn
```
and just monkey-patch it. Just do this at the top of each file that needs a logger. Easy and transparent.